client/components/OrderStage/OrderStage.jsx

```diff
@@ -3,23 +3,50 @@ import React from 'react';
 import './OrderStage.scss';
 
 function OrderStage(props) {
-  const { title, time, orders, id } = props;
+  const { title, orders, id, onDelete } = props;
+  const time = Math.max(...orders.map((order) => order.time));
+  const diff = Math.floor((Date.now() - time) / 1000);
+
+  const intervals = [
+    [3600, ' hours'],
+    [60, ' minutes'],
+    [10, '0 seconds'],
+  ];
+  const stringParams = intervals.reduce(
+    (accumulator, currentValue) =>
+      currentValue[0] <= accumulator[0] && accumulator[1] === 'just now'
+        ? currentValue
+        : accumulator,
+    [diff, 'just now']
+  );
+
+  const timeString =
+    stringParams[1] === 'just now'
+      ? stringParams[1]
+      : `${Math.floor(diff / stringParams[0])}${stringParams[1]} ago`;
 
   const counts = {};
 
   orders.forEach((elem) => {
     counts[elem.item._id] = (counts[elem.item._id] || 0) + 1;
   });
 
+  const getCallbackById = (orderId) => () => onDelete(orderId);
```

Comment:
@IgorChaikin This part creates new function for `onClick` handler for each order, is it possible to have only on `onClick` handler which figures out, which order._id should be deleted?